aten/src/ATen/native/transformers/cuda/attention_backward.cu

```diff
@@ -247,6 +250,8 @@ std::tuple<at::Tensor, at::Tensor, at::Tensor> _efficient_attention_backward(
     p.grad_key_ptr = (scalar_t*)grad_k.data_ptr();
     p.grad_value_ptr = (scalar_t*)grad_v.data_ptr();
     p.delta_ptr = (float*)delta.data_ptr();
+    p.scale = scale.has_value() ? 1.0f / scale.value()
```

Comment:
It seems easier to not make scale optional and do this ternary dance everywhere

Reply:
Okay figured out why the dance,
Scale needs to be the last arg, but this would come after args with default args. So need a good default hence -> None